gmail/modules/listener/http_service.bal

```diff
@@ -55,4 +89,212 @@ service class HttpService {
         }
         check caller->respond(http:STATUS_OK);
     }
+
+    isolated function dispatch(gmail:History history) returns @tainted error? {
```

Comment:
For modularity, i would like to keep all dispatch methods in a single class called dispatcher. Shall we keep it as it is? Why do we need to bring all methods here?

Reply:
Earlier it was possible to have a separate `class` for dispatching.
But now after making the `HttpService service class` as isolated, i am unable to use the previous approach.
So i had to move them here and do small changes

Reply:
<img width="882" alt="e1" src="https://user-images.githubusercontent.com/32201965/126322283-13b67597-1636-46ae-87a4-5fbfd1dc682e.png">

@abeykoon I tried to implement as you suggested. But i am having a blocking issue as shown in the above screenshot.

It gives compilation error as follows

`ERROR [http_service.bal:(84:56,84:63)] invalid attempt to transfer a value into a 'lock' statement with restricted variable usage
error: compilation contains errors`

This is why, i had to move the dispatch methods to this class